images/ubuntu-bun-ai/Dockerfile

```
@@ -10,17 +10,20 @@ ARG USERNAME=root
 # Elevate to root to install personal taste additions
 USER root
 
-# 1. Install CodeRabbit CLI
+# 1. Install system cron
+RUN apt-get update && apt-get install -y cron && rm -rf /var/lib/apt/lists/*
+
+# 2. Install CodeRabbit CLI
 RUN curl -fsSL https://cli.coderabbit.ai/install.sh | bash \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)